src/provider/view/utils/Markdown.ts

```diff
@@ -0,0 +1,16 @@
+const markdown = require('markdown-it');
+
+const md = markdown('zero', {breaks: true});
+
+md.enable('hr');
+md.enable('paragraph');
+md.enable('newline');
+md.enable('heading');
+md.enable('lheading');
+md.enable('list');
+md.enable('blockquote');
+md.enable('emphasis');
+
+export function renderMarkdown(str: string) {
```

Comment:
Update `client/index.ts` comments on `NotificationOptions` to indicate markdown support

Reply:
Also, suggest add tests to `createCenter/createToast.inttest.ts` that create a notification with markdown in the body, to test that that's working as expected

Reply:
Added documentation https://github.com/HadoukenIO/notifications-service/pull/105/commits/b84563735a9839cedc8833b3a08b8f658578676c